docs/contributing/documentation-guidelines/index.md

```diff
@@ -19,3 +21,36 @@ In terms of style, you should refer to the [Google developer documentation style
 - [Use sentence case](https://developers.google.com/style/capitalization) for document titles and section headings.
 - [Use descriptive link text](https://developers.google.com/style/link-text).
 - [Write short sentences](https://developers.google.com/style/translation#write-short,-clear,-and-precise-sentences) that are easy to understand and translate.
+
+## How to preview your modification on Autoware Documentation
+
+There are two ways to preview your modification on Autoware Documentation.
```

Comment:
The same reason as https://github.com/autowarefoundation/autoware-documentation/pull/203/files#r982397869.

```suggestion
There are two ways to preview your modification on a documentation site.
```

Reply:
Resolved https://github.com/autowarefoundation/autoware-documentation/pull/203/commits/ff49a125b82682e0d062b355a2a4a3258451dcd9